packages-ts/gauntlet-terra-contracts/src/commands/tooling/upload.ts

```diff
@@ -42,20 +43,29 @@ export default class UploadContractCode extends TerraCommand {
 
     const contractReceipts = {}
     const responses: any[] = []
+    const parsedRetryCount = parseInt(this.flags.maxRetry)
+    const maxRetry = parsedRetryCount ? parsedRetryCount : 5
     for (let contractName of askedContracts) {
       const contract = await getContract(contractName, this.flags.version)
       console.log('CONTRACT Bytecode exists:', !!contract.bytecode)
-      try {
-        const res = await this.upload(contract.bytecode, contractName)
+      for (let retry = 0; retry < maxRetry; retry++) {
+        try {
+          const res = await this.upload(contract.bytecode, contractName)
 
-        logger.success(`Contract ${contractName} code uploaded succesfully`)
-        contractReceipts[contractName] = res.tx
-        responses.push({
-          tx: res,
-          contract: null,
-        })
-      } catch (e) {
-        logger.error(`Error deploying ${contractName} code: ${e.message}`)
+          logger.success(`Contract ${contractName} code uploaded succesfully`)
+          contractReceipts[contractName] = res.tx
+          responses.push({
+            tx: res,
+            contract: null,
+          })
+        } catch (e) {
+          logger.error(`Error deploying ${contractName} Attempt ${retry + 1} Code: ${e.message}`)
```

Comment:
Apparently there are some uploads that succeed. Could we give a more explanatory error message?
Also, could we ask confirmation from the user before every upload? (`await prompt(...)`)

Reply:
What other information would you like in the error message? It currently gives the contract, the number of times retried so far and the error message from the request. I really don't have any other information to put unless you want me to do some kind of swtich on the message from the request with different messaging based on whether it is a 400 or 404 or 502 or something like that.

I will get the prompt added.

Reply:
Changed the wording a bit and added the prompt.